src/taipy/core/_entity/_migrate.py

```diff
@@ -322,16 +331,15 @@ def _load_all_entities_from_mongo(
 
 def __write_entities_to_mongo(
     _entities: Dict,
-    hostname: str = "localhost",
-    port: int = 27017,
-    user: str = "taipy",
-    password: str = "password",
+    hostname: str,
+    port: int,
+    user: str,
+    password: str,
```

Comment:
We should have as a default value the default value of the configuration.
What do you think?